### DISCUSSION TITLE
[BUG] Not possible to select which PV to be picked by recurring backup job

### DISCUSSION THREAD

**Discussion by ammirator-administrator** [longhorn/longhorn#8558] [General]
After reading the docs it is not fully clear to me if there indeed is a way to select only the needed PV's to create a backup
Let's say I have 3 PV one is for postgres, one for redis and one for minio

How do I make sure that my recurring job creates backups only for redis and postgres, let's say that I do not want any for minio
is that possible?
Maybe there are some selectors or something like that?

**Comment by innobead** [comment]
Converted this into a discussion first, unless it becomes a real issue later.

@c3y1huang can you help with this? Thanks.

**Comment by ammirator-administrator** [comment]
Hey @innobead , @c3y1huang 
Any news on this
Is there a possible way to select the PV for a backup?

**Reply by c3y1huang** [reply]
The recurring job can be applied to the Longhorn volume or the [PVC](https://longhorn.io/docs/1.6.1/snapshots-and-backups/scheduling-backups-and-snapshots/#with-persistentvolumeclaim-using-the-kubectl-command).

Is there a specific requirement or reason the recurring job should be working with the PV instead?

**Reply by ammirator-administrator** [reply]
Oh I see
No is just a terminology mistake from my side

So just to double check if I have 2 PVC lets say
- pg-data-pvc
- redis-data-pvc

And add this label only to first one `recurring-job-group.longhorn.io/<RECURRING-JOB-GROUP-NAME>=enabled`
Then the recurring job will do backups only for first one and skip second is that correct?
Also that means that I should use a different group name than `default` for recurring job

**Reply by ammirator-administrator** [reply]
That worked thanks
